src/Avatar/Avatar.js

```diff
@@ -89,6 +89,7 @@ class Avatar extends React.PureComponent {
             presenceType: presence,
             clickable: !!onClick,
             fade: fadeIndication,
+            text: !!text,
```

Comment:
I would rename to the following to clarify that's a boolean and not the actual text:
```suggestion
            hasText: !!text,
```